examples/scripts/stable_diffusion_tuning.py

```diff
@@ -0,0 +1,160 @@
+import os
```

Comment:
(nit) can you add the copy right here with your GH handle + Hugging Face team: https://github.com/lvwerra/trl/blob/main/trl/import_utils.py#L1